npc/re/jobs/novice/academy.txt

```diff
@@ -13,80 +13,226 @@
 //= 1.0.0 Initial Release. [Kisuka]
 //= 1.0.1 Converted item constants to IDs and added hair coupon. [Kisuka]
 //= 2.0.0 Converted from Hercules Script to rAthena Script. [Capuche]
+//= 2.0.1 Add new Intro Renewal based on Hercules [Jenkijo]
 //==============================================================================
 
-// - Captain Carocc (Ship)
-// -----------------------------------------------------------------------------
-iz_int,100,91,3	script	Captain Carocc#iz_int	4_M_REINDEER,{
-	if (Class == Job_Novice) {
-		cutin "fly_trock", 2;
-		if (!isbegin_quest(7471)) {
-			mes "[Captain Carocc]";
-			mes "Had a good dream?";
-			mes "Soon, we will get to ^4d4fffIzlude^000000.";
-			next;
-			mes "[Captain Carocc]";
-			mes "And you can talk to other people like you just talked to me.";
-			next;
-			cutin "tutorial01",3;
-			mes "!- Information -!";
-			mes "^4d4fffLeft mouse click allows basic game controls.";
-			mes "Such as conversation with NPC, movement, and attacks.^000000";
-			next;
-			cutin "fly_trock",2;
-			mes "[Captain Carocc]";
-			mes "Many youngsters visit the ^4d4fffRune Midgard Kingdom^000000 to chase their dreams.";
-			mes "Izlude is a satellite city of Prontera, the capital of the Rune Midgard Kingdom.";
-			next;
-			mes "[Captain Carocc]";
-			mes "Izlude always welcomes young adventurers like you who have just started their journey to the World.";
-			mes "You are one of them of course.";
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
+		specialeffect2 EF_ANGEL3; // On official it is some kind of Poring Angel, but I can't find it
+		navigateto("int_land", NAV_NONE, 1); // individual map name not required
+		izintspawn = 1;
+	}
+	end;
+}
+
+//== Wounded Swordsman (iz_int) =================================
+iz_int,56,32,3	script	Wounded Swordsman#izint	4_TOWER_01,8,8,{
+	if (!@display) {
+		classchange 4_TOWER_02;
+		npctalk "Ohhhhhhh....";
+		@display = 1;
+		end;
+	}
+	if(checkquest(21001) == -1) {
+		mes "[Wounded]";
+			mes "Wow! Thank you a lot!";
+			mes "I don't know how this happend to our ship";
+			mes "but we should go to see the captain.";
 			next;
-			mes "[Captain Carocc]";
-			mes "I would like to tell you more stories, but it's time to get ready for arrival.";
-			mes "There is an academy in Izlude for newcomers like yourself.";
+			mes "[Wounded]";
+			mes "... ohh, it seems my body is injured too much.";
+			mes "Maybe you can go without me?";
+			setquest(21001);
+			close;
```

Comment:
The indentation for the above lines could be move down one to match the NPC header name.